src/sider/1-forside/seksjoner/kategorier/Kategorimeny/Kategorimeny.tsx

```diff
@@ -69,6 +61,21 @@ class Kategorimeny extends Component<MergedProps, State> {
         aktiveKategorier={aktiveKategorier}
       />
     );
+    return <>
+      <Helmet>
+        <title>
+          {sideTittel(
+          `${localeTekst(valgtKategori.tittel, locale)} - ${intl.formatMessage({
+          id: "tittel.soknader"
+        })}`)}
+        </title>
+        <meta name="description"
+              content={intl.formatMessage(
+                {id: 'kategori.meta_desc'},
+                {kategoritittel: localeTekst(valgtKategori.tittel, locale)})} />
+      </Helmet>
```

Comment:
Kan vurdere å ha dette et annet sted enn i meny-komponenten, men i og med at det lå der fra før av trenger vi ikke å flytte det for denne PRs skyld